packages/ckeditor5-remove-format/tests/integration.js

```diff
@@ -9,6 +9,7 @@ import RemoveFormat from '../src/removeformat';
 import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
 import Heading from '@ckeditor/ckeditor5-heading/src/heading';
 import Image from '@ckeditor/ckeditor5-image/src/image';
+import ImageResizeEditing from '@ckeditor/ckeditor5-image/src/imageresize/imageresizeediting';
```

Comment:
```suggestion
import ImageResize from '@ckeditor/ckeditor5-image/src/imageresize';
```

Let's just add full featured image resize editing plugin like you did in another manual test.